Enable GTK_WIDGET macros

This patch enables the macros that clutter-gtk is still using

Signed-off-by: Saul Wold <sgw@linux.intel.com>

Index: git/clutter-gtk/Makefile.am
===================================================================
--- git.orig/clutter-gtk/Makefile.am	2011-01-30 13:10:29.000000000 +0800
+++ git/clutter-gtk/Makefile.am	2011-01-30 13:15:09.000000000 +0800
@@ -15,8 +15,6 @@
 	-DCLUTTER_GTK_COMPILATION	\
 	-DG_DISABLE_DEPRECATED		\
 	-DGDK_PIXBUF_DISABLE_DEPRECATED	\
-	-DGDK_DISABLE_DEPRECATED	\
-	-DGTK_DISABLE_DEPRECATED	\
 	-DCLUTTER_DISABLE_DEPRECATED	\
 	$(CLUTTER_GTK_DEBUG_CFLAGS) 	\
 	$(NULL)
